modules/rpc/handler.go

```diff
@@ -30,6 +30,8 @@ import (
 	"go.uber.org/yarpc/api/transport"
 )
 
+const panicResponse = "Server Error"
```

Comment:
1. ya, I want to through a same general error msg for panic. do we need to share this const between `rpc` and `uhttp`
2. sure